health.go

```diff
@@ -80,6 +81,7 @@ func NewDefaultHealthCheck() *HealthCheck {
 		GracePeriodSeconds:     30,
 		IntervalSeconds:        10,
 		TimeoutSeconds:         5,
+		DelaySeconds:           15,
```

Comment:
yes, but we would like to be able to change it. And we could have some app definitions that include delaySeconds and we would like to support them.